.env.example

```
@@ -264,6 +264,8 @@ LLM_VERTEX_OPENAI_REGION=global
 # Paste the full GCP service account JSON as a single-line string.
 # Project ID is extracted automatically. See docs/integrations/vertex-anthropic for setup.
 LLM_VERTEX_ANTHROPIC_SERVICE_ACCOUNT_JSON={"type":"service_account","project_id":"your-project","private_key":"-----BEGIN RSA PRIVATE KEY-----\n...\n-----END RSA PRIVATE KEY-----\n","client_email":"vertex-ai-caller@your-project.iam.gserviceaccount.com","token_uri":"https://oauth2.googleapis.com/token"}
+# Only needed to call a different project than the service account's own.
+LLM_VERTEX_ANTHROPIC_PROJECT=
```

dotenv-linter (4.0.0)

[warning] [UnorderedKey] The LLM_VERTEX_ANTHROPIC_PROJECT key should go before the LLM_VERTEX_ANTHROPIC_SERVICE_ACCOUNT_JSON key

(UnorderedKey)